docs/configurations.md

```diff
@@ -0,0 +1,270 @@
+To avoid frequent errors caused internally by [`SQLITE_BUSY`](https://www.sqlite.org/rescode.html#busy), we recommend setting a [`busy_timeout`](https://www.sqlite.org/c3ref/busy_timeout.html) parameter.
+
+## ScalarDB Server configurations
+
+[ScalarDB Server](scalardb-server.md) is a standalone server that provides a gRPC interface to ScalarDB.
+This section explains ScalarDB Server configurations.
+
+In addition to [transaction manager configurations](#transaction-manager-configurations) and [other configurations](#other-configurations), the following configurations are available for ScalarDB Server:
+
+| Name                                              | Description                                                                                      | Default                |
+|---------------------------------------------------|--------------------------------------------------------------------------------------------------|------------------------|
+| `scalar.db.server.port`                           | Port number for ScalarDB Server.                                                                 | `60051`                |
+| `scalar.db.server.prometheus_exporter_port`       | Prometheus exporter port. Prometheus exporter will not be started if a negative number is given. | `8080`                 |
+| `scalar.db.server.grpc.max_inbound_message_size`  | The maximum message size allowed to be received.                                                 | The gRPC default value |
+| `scalar.db.server.grpc.max_inbound_metadata_size` | The maximum size of metadata allowed to be received.                                             | The gRPC default value |
+| `scalar.db.server.decommissioning_duration_secs`  | The decommissioning duration in seconds.                                                         | `30`                   |
+
+For details about ScalarDB Server, see [ScalarDB Server](scalardb-server.md).
+
+## Other configurations
+
+This section explains other configurations.
+
+Other configurations are available for ScalarDB:
+
+| Name                                                             | Description                                                                                                                                                                                                       | Default              |
+|------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------|
+| `scalar.db.metadata.cache_expiration_time_secs`                  | ScalarDB has a metadata cache to reduce the number of requests to the database. This setting specifies the expiration time of the cache in seconds.                                                               | `-1` (no expiration) |
```

Comment:
This is not for this PR, but no expiration by default sounds a bit unsafe as, IIUC, other ScalarDB server instances keep old cache? 30 seconds by default or so might be safer